### PR TITLE
Ignore `time` related vulnerabilities

### PR DESCRIPTION
**Pull request by pvdrz**
Angry sec noises

### CODE REVIEW

File: .cargo/audit.toml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+[advisories]
```
Comment (thibault-martinez) [inline .cargo/audit.toml]: Any way to have this file in `.github/workflows/` ?

Comment (pvdrz) [reply .cargo/audit.toml]: i suppose that we could modify the workflows to move whatever is in `.github/workflows/.cargo` to `~/.cargo` but I'd have to investigate.

Comment (thibault-martinez) [reply .cargo/audit.toml]: For reference https://github.com/iotaledger/bee/blob/dev/.github/workflows/licenses.yml#L29

**Comment by Adam-Gleave** [comment]
Adding to #786.

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**